todoist/managers/sections.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+from .. import models
+from .generic import AllMixin, GetByIdMixin, Manager, SyncMixin
+
+
+class SectionsManager(Manager, AllMixin, GetByIdMixin, SyncMixin):
+
+    state_name = 'sections'
+    object_type = 'section'
+
+    def add(self, name, project_id, **kwargs):
```

Comment:
Yes, I'm not sure, I saw that we have this for projects/items, but they were legacy get function from `v5` or even older, bypassing the Sync API.  If no-one needs such a function, maybe we don't need it, as we also have the REST API now.

Reply:
It's what we suggest in the documentation. :thinking: 
We can change that, but currently it's the usual way of doing things:

```
>>> import todoist
>>> api = todoist.TodoistAPI('0123456789abcdef0123456789abcdef01234567')
>>> project = api.projects.get_by_id(128501815)
>>> project.delete()
>>> api.commit()
```

So I expected the same for sections.

Reply:
Hm, the `api.sections.get_by_id()` would work, but only if the data are synced.  So it seems that if we don't have the data locally then this will fail: https://github.com/Doist/todoist-python/blob/755b9bd8a4fdf4e96b2381613ac0c4bed99731e5/todoist/managers/generic.py#L40

So we need to add the `sections/get` endpoint, then.

Reply:
We now depend on this PR: https://github.com/Doist/Todoist/pull/1774